firefox-ios/Client/Assets/Images.xcassets/fingerprinterLarge.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "filename" : "✏️ Icon 24.pdf",
```

Comment:
Looking at the file name it doesn't seem this image was not downloaded from the Acorn repo. Please ensure that all standard images are taken directly from the Acorn repo.